velox/exec/tests/HashJoinTest.cpp

```diff
@@ -4670,6 +4672,86 @@ TEST_F(HashJoinTest, dynamicFiltersWithSkippedSplits) {
   }
 }
 
+TEST_F(HashJoinTest, dynamicFiltersAppliedToPreloadedSplits) {
+  vector_size_t size = 1000;
+  const int32_t numSplits = 5;
+  std::vector<RowVectorPtr> probeVectors;
+  probeVectors.reserve(numSplits);
+  FlagUpdater(FLAGS_split_preload_per_driver, 3);
```

Comment:
Are these other tests that use split_preload_per_driver flag and therefore need to be updated to use the new RAII class? If so, would you extract the RAII into a separate PR and updating existing tests?

Reply:
Got it. Let's make a new PR to replace this gflag with a configuration property. Then come back to this PR.

Reply:
https://github.com/facebookincubator/velox/pull/7931, thanks.